quick-start/examples/n5.min.js

```diff
@@ -0,0 +1,21 @@
+import { Scene, Circle, Rectangle } from "https://unpkg.com/pencil.js/dist/pencil.esm.js";
+
+const scene = new Scene(undefined, {
+    fill: "#45a8ff",
+});
+
+const sun = new Circle(scene.center, 100, {
+    fill: "orange",
+});
+
+const width = scene.width;
+const height = scene.height / 2;
+// Positions can be declared with a Position object or an array
```

Comment:
```suggestion
// Positions can be declared with a Position object or an array containing a x- and y-coordinate`
```
If you wish to be extremely beginner-friendly, I would add this explanation.